### PR TITLE
[3469582] Fixed Alert content type - Page visibility validation issue

### PR DESCRIPTION
**Pull request by joshua-salsadigital**
https://www.drupal.org/project/civictheme/issues/3469582

## Checklist before requesting a review

- [x] I have formatted the subject to include the issue number
  as `[#123] Verb in past tense with a period at the end.`
- [x] I have provided information in the `Changed` section about WHY something was
  done if this was a bespoke implementation.
- [x] I have performed a self-review of my code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have run new and existing relevant tests locally with my changes,
  and they have passed.
- [ ] I have provided screenshots, where applicable.

## Changed

1. Fixed Alert content type - Page visibility validation issue
2. Replaced `<br /> `with ``.

### CODE REVIEW

File: components/03-organisms/alert/alert.js [hunk 1]
```diff
@@ -85,6 +85,8 @@ CivicThemeAlert.prototype.isVisible = function (visibilityString) {
   }
 
   let pageVisibility = visibilityString.replace(/\*/g, '[^ ]*');
+  // Replace '<br />' with ''.
+  pageVisibility = pageVisibility.replace(new RegExp('<br />', 'g'), '');
```
Comment (joshua-salsadigital) [inline components/03-organisms/alert/alert.js:89]: Required as JSON.parse add/converts line break to <br />

File: components/03-organisms/alert/alert.js [hunk 2]
```diff
@@ -85,6 +85,8 @@ CivicThemeAlert.prototype.isVisible = function (visibilityString) {
   }
 
   let pageVisibility = visibilityString.replace(/\*/g, '[^ ]*');
+  // Replace '<br />' with ''.
+  pageVisibility = pageVisibility.replace(/<br \/>/g, '');
```
Comment (joshua-salsadigital) [inline components/03-organisms/alert/alert.js:89]: Required as JSON.parse add/converts line break to

**Comment by joshua-salsadigital** [comment]
Not sure if we need to add code coverage for all, as i ran it locally and see that, lot of components are not covered. 
Screenshot 2024-08-22 at 2 14 44 PM

**Review by richardgaunt — CHANGES_REQUESTED** [review]
I would prefer this handled in Drupal theme as its a case of Drupal adding `br` tags to line endings in the JSON output

**Pull request closed without merging**